explore/visualization/README.md

```diff
@@ -0,0 +1,9 @@
+# A directory for creating Vega based visualizations
+
+In order to run these sample files, you should first start up a simple HTTP server such as:
+
+```sh
+python -m SimpleHTTPServer 8000

```

Comment:
Note that this repository's environment uses Python 3.

In Python 3, I think this should be:

```sh
python -m http.server 8000
```

Feel free to include both commands, if you'd like.